example/index.js

```diff
@@ -0,0 +1,41 @@
+// IIFE - Immediately Invoked Function Expression
+(function (code) {
+
+    // The global jQuery object is passed as a parameter
+    code(window.jQuery, window, document)
+
+}(function ($, window, document) {
+
+    // The $ is now locally scoped
+    // Listen for the jQuery ready event on the document
+    $(async function () {
+
+    })
+
+    $("#viewAsset").click(function(){
```

Comment:
let's use arrow notation instead of function notation. (you use arrow notation below.) arrow notation preserves the surrounding value of `this`